packages/datagateway-dataview/public/res/default.json

```diff
@@ -10,7 +10,7 @@
     "open_data_warning": {
       "message": "Open data",
       "tooltip": "Only open data is displayed when browsing anonymously. Please login to see embargoed data.",
-      "studies_tooltip": "Only open data is currently supported by the experiment view. Please browse via facility cycle to view closed data",
+      "experiments_tooltip": "Only open data is currently supported by the experiment view. Please browse via facility cycle to view closed data",
```

Comment:
I prefer leaving any programmatic references to study as Study - as that equates to the ICAT entity. As otherwise the language file is inconsistent as there's other references to Studies

Reply:
I guess that's fair, allows for more consistency. I'll revert that change.